pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "nzsl-signbank"
+version = "0.1.0"
+description = "Web based database for sign language lexicons and corpuses. Fork of NGT-signbank."
+authors = ["Ackama <info@ackama.com>"]
+license = "BSD-3-Clause license"
+readme = "README.md"
+homepage = "https://signbank.nzsl.nz/"
+repository = "https://github.com/ODNZSL/NZSL-signbank"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+django = "3.2.25"
```

Comment:
`django = "~3.2.25"

I just think checking which of these dependencies really need to be pinned as exact versions would be good with this 😄

Reply:
True. I just copied the initial ones across from `requirements.txt` so this would work as-is.
I can definitely try unpinning stuff.